we use the functions provided by these macros, so make sure we call them

http://bugs.gentoo.org/218666

--- configure.ac
+++ configure.ac
@@ -31,6 +31,8 @@
 SHARED_VERSION_INFO="1:17:0"
 
 AC_PROG_CC
+AM_PROG_CC_C_O
+AC_PROG_CXX
 AM_PROG_LIBTOOL
 
 AC_CHECK_PROG(autogen, autogen, yes, no)
